packages/terra-core-docs/CHANGELOG.md

```diff
@@ -2,15 +2,27 @@
 
 ## Unreleased
 
+* Changed
+  * Updated Arrange component example to use `span` element instead of `div`.
+
+* Added
+  * Added test example for controlled form native select component.
+  * Added more realistic examples for terra-toggle-button.
+
+## 1.29.0 - (June 28, 2023)
+
 * Changed
   * Updated `terra-status-view - Change Variant` example component to use the apply button to change the selected variant.
-  
+
+* Added
+  * Added documentation updates for `Single Select` in `terra-form-select`.
+
 ## 1.28.0 - (June 14, 2023)
 
 * Added
   * Added terra-scroll A11y tests.
   * Added an example in terra-content-container without interactive elements.
-
+  * Added examples for terra-content-container with dark colors.
```

Comment:
Same as: https://github.com/cerner/terra-core/pull/3831/files#r1261701315

Reply:
Rebased with main and now the file is up to date.